### PR TITLE
Update Rails to 5.1.2 and it's friends

### PR DESCRIPTION
**Pull request by timurvafin**

### CODE REVIEW

File: Guardfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ guard :rspec, cmd: "bin/rspec" do
   dsl.watch_spec_files_for(ruby.lib_files)
 
   # Rails files
-  rails = dsl.rails(view_extensions: %w(slim))
+  rails = dsl.rails(view_extensions: %w[slim])
```
Comment (dmitrytrager) [inline Guardfile]: new preferred syntax?

Comment (timurvafin) [inline Guardfile]: yep, looks like our friend @shir forced this change here https://github.com/bbatsov/rubocop/issues/4039

**Comment by timurvafin** [comment]
Going to ignore CC fails b/c of https://github.com/bbatsov/rubocop/issues/4039

File: Guardfile (hunk 1 above)
Comment (artembiserov) [inline Guardfile]: https://github.com/bbatsov/ruby-style-guide#percent-i
`Apply this rule only to arrays with two or more elements `

**Pull request merged**